### DISCUSSION TITLE
Create child page

### DISCUSSION THREAD

**Discussion by robkle153** [google/docsy#1636] [Q&A]
When you want to create a child page, it automatically creates a file as child from the page where you clicked, but the _index.md file (p.e. you want to create a new blog post) becomes a folder. 

I get something like this: .../-/new/testbranch/content/en/Blog/_index.md/new-page.md

My blog _index.md should stay, only a new page should be created.

Ask for help, thank you guys!

**Comment by deining** [comment]
Hi @robkle153,

I'm trying to help you out. Seems like you are new to hugo. You have to familiarize yourself with hugo's concept of [sections]() and probably also [page bundles](https://gohugo.io/content-management/page-bundles/). Once you understood these concepts and know how to create a section tree (inside the folder `content`), you can successfully manage your contents to be published.

Back to your question: is is not fully clear to me what you are trying to achieve, but most likely, you need a structure like this:

```
content
└── blog        <-- Section, because first-level dir under content/
    ├── _index.md       // https://yoursite.tld/blog
    └── new-page.md     // https://yoursite.tld/blog
````

Does this help?

**Reply by robkle153** [reply]
Thank you for your reply. I would like to use the links on the right side menu at Docs/Blog to create a new child page. 
I followed the steps in the userguide but when redirecting to the git repo I get something like in the picture. 

![image](https://github.com/google/docsy/assets/139347669/ad75f7b3-f82a-4d70-8e32-d38486b0378b)

It overwrites the _index.md file and creates a folder from it.

![image](https://github.com/google/docsy/assets/139347669/81af716b-969d-4225-870b-130eb55102be)
![image](https://github.com/google/docsy/assets/139347669/c86d16bf-0e9a-4ef4-b60f-8cfd95b7e99c)

**Comment by deining** [comment]
> Thank you for your reply. I would like to use the links on the right side menu at Docs/Blog to create a new child page.

Now I understand what you are trying to do.

> I followed the steps in the userguide but when redirecting to the git repo I get something like in the picture.
> It overwrites the _index.md file and creates a folder from it.

You hit a bug in Docsy theme. Can you open a [new issue](https://github.com/google/docsy/issues/new) for that, please?

**Reply by robkle153** [reply]
I opened a new issue :) Thanks for helping me
